src/Stripe.net/Services/Invoices/InvoiceFinalizeInvoiceOptions.cs

```diff
@@ -1,11 +1,8 @@
 namespace Stripe
 {
-    using System;
-    using System.Collections.Generic;
     using Newtonsoft.Json;
-    using Stripe.Infrastructure;
 
-    public class InvoiceFinalizeOptions : BaseOptions
+    public class InvoiceFinalizeInvoiceOptions : BaseOptions
```

Comment:
I'm still on the fence about that one. I wonder if we shouldn't just go back to `InvoiceFinalizeOptions` and rename the method `Finalize` (and same for all the others)